docs/ECLLanguageReference/ECLR_mods/BltInFunc-ORDERED.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="ORDERED">
+  <title>ORDERED</title>
+
+  <para><emphasis role="bold">[</emphasis><emphasis>attributename</emphasis>
+  :=<emphasis role="bold"> ] ORDERED<indexterm>
+      <primary>ORDERED</primary>
+    </indexterm><indexterm>
+      <primary>ORDERED function</primary>
+    </indexterm>(</emphasis><emphasis> actionlist </emphasis><emphasis
+  role="bold">)</emphasis></para>
+
+  <para><informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec colwidth="82.95pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis>attributename</emphasis></entry>
+
+            <entry>Optional. The action name, which turns the action into an
+            attribute definition, therefore not executed until the
+            attributename is used as an action.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis>actionlist</emphasis></entry>
+
+            <entry>A comma-delimited list of the actions to execute in order.
+            These may be ECL actions or external actions.</entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable></para>
+
+  <para>The <emphasis role="bold">ORDERED </emphasis>action executes the items
+  in the <emphasis>actionlist</emphasis> in the order in which they appear in
+  the <emphasis>actionlist</emphasis>. This is useful when a subsequent action
+  requires the output of a precedent action. </para>
```

Comment:
It isn't really useful for this case - that's what sequential is for.  It is useful for the case where actions need to be performed in order, but the actions aren't dependent on each other.

The difference between ORDERED and SEQUENTIAL is that if the actions share common values they will only be evaluated once if ORDERED is used, but multiple times if SEQUENTIAL is used.